plottable.css

```diff
@@ -65,17 +65,17 @@ svg.plottable {
 }
 
 .plottable .label-area text {
-  fill: #32313F;
+  fill: white;
   font-family: "Helvetica Neue", sans-serif;
   font-size: 14px;
 }
 
-.plottable .light-label text {
```

Comment:
Renaming the CSS classes is technically API break, so we have to leave them as-is. You can still add the other class, but I'd leave the original one.